### PR TITLE
translate_ruby_and_rails_to_english

### PR DESCRIPTION
**Pull request by KhanhLD**

### CODE REVIEW

**Comment by nguyenducgiang** [comment]
OK.

File: eng/rails/gems.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+* [cucumber-rails](https://github.com/cucumber/cucumber-rails) - A helpful tool to implement integration test in Rails. Using it allow us to use Cucumber in Rails.
+
+* [factory_girl](https://github.com/thoughtbot/factory_girl) - Help creating objects for test actions.
+
+* [ffaker](https://github.com/EmmanuelOga/ffaker) - Help creating sample data easily.
+
+* [guard](https://github.com/guard/guard) - Can monitoring changes made to files and do automated tasks. Used in many tools.
+
+* [spork](https://github.com/sporkrb/spork) - A DRb server for testing frameworks such as RSpec or Cucumber. By pre-populating test environemt, it helps shorten time spent on testing.
```
Comment (octoberstorm) [inline eng/rails/gems.md]: typo `environemt`

Comment (octoberstorm) [inline eng/rails/gems.md]: otherwise LGTM

**Comment by octoberstorm** [comment]
OK

**Comment by linhnt** [comment]
ok

**Pull request merged**